### PR TITLE
React Capstone

### PR DESCRIPTION
**Pull request by leolpaz**
### React-Capstone 

Added on this pull request:

- One page with a list of items that can be filtered
- Page for item details accessible by clicking on the item on the home page
- CSS following the design guidelines for mobile
- Slightly different CSS for desktop version as per one of the additional requirements
- Back button returns back to the home page
- Input based filter using regex
- Unique routes for each page
- Unit tests for pure functions on redux code
- Integration test for user flow on the page
- Project deployment on Netlify

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,49 @@
...
-### Deployment
+- Clone the repository with git clone https://github.com/leolpaz/React-Capstone.git;
+- Install dependencies with npm i/yarn install;
+- Use npm start/yarn start to create the dev server;
+- Go to http://localhost:3000/ on your browser if it doesn't do it automatically once you create the dev server;
+- Run "npm/yarn run test" to see if the tests are passing, everything should pass;
 
 
 
 ## Authors
```
Comment (rloterh) [inline README.md]: - [x] Kindly change the header title _Authors_ to _Author_, as this project is done solely.

File: src/components/Nav.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    if (valid === true) {
+      return pathname.slice(1, pathname.length);
+    }
+    return placeholder;
+  };
+
+  return (
+    <div className="nav d-flex justify-content-between">
+      <Link data-testid="back-arrow" to="./"><AiOutlineArrowLeft /></Link>
```
Comment (rloterh) [inline src/components/Nav.js]: - [x] Kindly ensure that the _back_ navigation button is not shown on the _Homepage_. This button performs no function on the _Homepage_.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Leo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to complete this project but you are almost there! 🚦  🚦  

## Good Points 👍 
- Descriptive Pull Request
- Nice to acknowledge [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
- App is deployed and running correctly
- Good video presentation of the project
- Design matches provided template
- Redux is effectively used to manage state data
- App is running as intended
- Test cases are passing
- Highly organized code


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Nav.js (hunk 2 above)
Comment (leolpaz) [reply src/components/Nav.js]: Thanks for the review!

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**